### PR TITLE
[16.0][OU-ADD] google_drive: Nothing to do

### PR DESCRIPTION
**Pull request by daiduongnguyen-odoo**
Deprecated module, this module is no longer present as its'feature completely removed from odoo due to change in OOB flow check PR `https://github.com/odoo/odoo/pull/94703`

### CODE REVIEW

File: docsource/modules150-160.rst [hunk 1]
```diff
@@ -156,7 +156,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | google_calendar                                 |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| google_drive                              |                      |                                                 |
+| |del| google_drive                              |Nothing to do         |Deprecated module, features completely removed   |
```
Comment (daiduongnguyen-odoo) [inline docsource/modules150-160.rst]: Can i write like this XD

**Comment by pedrobaeza** [comment]
In this case, you can't say "Nothing to do", as there's no replacement. Maybe the other description. This should be manually removed IMO.

**Comment by daiduongnguyen-odoo** [comment]
> In this case, you can't say "Nothing to do", as there's no replacement. Maybe the other description. This should be manually removed IMO.

OK , i see, maybe i'll go with `Deprecated Module` instead

File: docsource/modules150-160.rst [hunk 2]
```diff
@@ -156,7 +156,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | google_calendar                                 |Done                  |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| google_drive                              |                      |                                                 |
+| |del| google_drive                              |Deprecated module     |Obsolete module, features completely removed     |
```
Comment (MiquelRForgeFlow) [inline docsource/modules150-160.rst]:
```suggestion
| |del| google_drive                              | Deprecated module    | Obsolete module                                  |
```
I prefer it shorter.

Comment (daiduongnguyen-odoo) [reply docsource/modules150-160.rst]: Same like https://github.com/OCA/OpenUpgrade/pull/4002

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**